src/WorkOS.net/Services/UserManagement/Entities/AuthorizedOrganizations.cs

```diff
@@ -0,0 +1,16 @@
+namespace WorkOS
+{
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Organizations user has authorized access to.
+    /// </summary>
+    public class AuthorizedOrganizations
```

Comment:
Seems like this should be named in the singular, as it only contains a single organization?

```suggestion
    public class AuthorizedOrganization
```